cmd/cluster-kube-controller-manager-operator/render/render.go

```diff
@@ -25,16 +28,18 @@ type renderOpts struct {
 	manifest genericrenderoptions.ManifestOptions
 	generic  genericrenderoptions.GenericOptions
 
-	disablePhase2 bool
-	errOut        io.Writer
+	clusterConfigFile string
+	disablePhase2     bool
+	errOut            io.Writer
 }
 
 // NewRenderCommand creates a render command.
 func NewRenderCommand(errOut io.Writer) *cobra.Command {
 	renderOpts := &renderOpts{
-		manifest: *genericrenderoptions.NewManifestOptions("kube-controller-manager", "openshift/origin-hyperkube:latest"),
-		generic:  *genericrenderoptions.NewGenericOptions(),
-		errOut:   errOut,
+		manifest:          *genericrenderoptions.NewManifestOptions("kube-controller-manager", "openshift/origin-hyperkube:latest"),
+		generic:           *genericrenderoptions.NewGenericOptions(),
+		errOut:            errOut,
+		clusterConfigFile: "/assets/tectonic/99_openshift-cluster-api_cluster.yaml",
```

Comment:
would leave this out actually. Otherwise we keep installer details around here for ages.